darwin/PlatformHelpers.c

```diff
@@ -103,9 +103,8 @@ double Platform_calculateNanosecondsPerMachTick(void) {
     *    the "Apple M1" chip specifically when running under Rosetta 2.
     */
 
-   size_t cpuBrandStringSize = 1024;
-   char cpuBrandString[cpuBrandStringSize];
-   Platform_getCPUBrandString(cpuBrandString, cpuBrandStringSize);
+   char cpuBrandString[1024] = "";
+   Platform_getCPUBrandString(cpuBrandString, sizeof(cpuBrandString));
```

Comment:
Technically not VLA but this change helps readability anyway :)

Reply:
Um, actually … ;-) Although this uses a constant number (which the compiler can deduce), syntactically it <s>is</s>was.

BTW: I did a `grep --color=always -nRP '\b(?!return)\w+ \w+\[[^\]0-9A-Z_]'|less -R` through the source and had a look at the remaining locations found by this. Quite a nice way to quickly get through the source without straight up working with the AST …